civihr_default_theme/scss/base/_mixins.scss

```diff
@@ -181,3 +181,9 @@
   background-repeat: no-repeat;
   background-position: right center;
 }
+
+@mixin views-label {
```

Comment:
why do we need this new mixin?

Reply:
I want to mimic only certain properties of a views label, and I need this in many places.